src/Microsoft.Health.SqlServer/Features/Schema/Manager/SchemaClient.cs

```diff
@@ -27,7 +27,7 @@ public SchemaClient(IHttpClientFactory httpClientFactory)
 
     public void SetUri(Uri uri)
     {
-        if (_httpClient.BaseAddress == null)
+        if(_httpClient.BaseAddress == null)
```

Comment:
nit: missing space

Reply:
Thanks for noticing this; fixed it in my latest commit.